en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+| <span id="PARACHUTE_STATUS"></span>[PARACHUTE_STATUS](../messages/common.md#PARACHUTE_STATUS)                                           | Current status of the parachute. Recommended to publish this at a regular rate.  
+| <span id="MAV_CMD_SET_PARACHUTE_ARM"></span>[MAV_CMD_SET_PARACHUTE_ARM](../messages/common.md#MAV_CMD_SET_PARACHUTE_ARM)                | Command to arm/disarm parachute module trigger sources.|
+
+| Enum Values                                                                                                              | Description               |
+| ------------------------------------------------------------------------------------------------------------------ | ------------------------- |
+| <span id="MAV_TYPE_PARACHUTE"></span>[MAV_TYPE_PARACHUTE](../messages/minimal.md#MAV_TYPE_PARACHUTE)                             | Type of the component (parachute).                   |
+| <span id="MAV_COMP_ID_PARACHUTE"></span>[MAV_COMP_ID_PARACHUTE](../messages/minimal.md#MAV_COMP_ID_PARACHUTE)                           | ID of the component (parachute).   
+
+
+| Enum                                                                                                               | Description               |
+| ------------------------------------------------------------------------------------------------------------------ | ------------------------- |
+| <span id="PARACHUTE_TRIGGER_FLAGS"></span>[PARACHUTE_TRIGGER_FLAGS](../messages/common.md#PARACHUTE_TRIGGER_FLAGS) | Parachute trigger sources.        |
+| <span id="PARACHUTE_DEPLOYMENT_TRIGGER"></span>[PARACHUTE_DEPLOYMENT_TRIGGER](../messages/common.md#PARACHUTE_DEPLOYMENT_TRIGGER) | Parachute deployment trigger source. |
+| <span id="PARACHUTE_SAFETY_FLAGS"></span>[PARACHUTE_SAFETY_FLAGS](../messages/common.md#PARACHUTE_SAFETY_FLAGS) | Parachute module safety-related flags. |
+| <span id="PARACHUTE_ERROR_FLAGS"></span>[PARACHUTE_ERROR_FLAGS](../messages/common.md#PARACHUTE_ERROR_FLAGS) | Fault/health indications. |
+
+## Implementation and Messages
+
+### Parachute Connection
+
+Parachutes are expected to follow the [Heartbeat/Connection Protocol](https://github.com/mavlink/mavlink-devguide/blob/master/en/services/heartbeat.md) and send a constant flow of heartbeats (nominally at 1Hz). Parachutes are identified via their type [MAV_TYPE_PARACHUTE](#MAV_TYPE_PARACHUTE).
+Individual parachutes are distinguished via their unique component ID, which by default should be [MAV_COMP_ID_PARACHUTE](#MAV_COMP_ID_PARACHUTE) (though this is not mandated and any ID may be used).
+
+Once a heartbeat is received, the drone can then send a [MAV_CMD_REQUEST_MESSAGE](https://mavlink.io/en/messages/common.html#MAV_CMD_REQUEST_MESSAGE) command to the parachute to receive information about its status. [MAV_CMD_SET_PARACHUTE_ARM](#MAV_CMD_SET_PARACHUTE_ARM) can also then be sent to configure the arming status of the various possible trigger mechanisms of the parachute. 
```

Comment:
since `PARACHUTE_STATUS` should be publishing at default rate by default, is the `MAV_CMD_REQUEST_MESSAGE` referring to getting info like software version from `COMPONENT_INFORMATION_BASIC`?

Reply:
I was referencing in general you can request any message related to parachute. Even though `PARACHUTE_STATUS` is publishing at a default rate, a user could request  the status or `COMPONENT_INFORMATION_BASIC` I will update so that part is more clear though